css/index.css

```diff
@@ -1 +1,315 @@
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+html {
+font-size:62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom:15px;
+}
+
+p{
+    font-size: 1.6rem;
+}
+
+.container {
+  width: 880px;
+  width:100%
+  /* TB LR */
+  margin: auto;
+  border: 1px solid red;
+  }
+
+/* header */
+header {
+    width:100%;
+    display: flex;
+    font-size: 1.6rem;
+}
+
+header nav { /* display: flex;*/
+ /* width: 610px;*/
+ width: 100%;
+ margin: 38px 15 0;
+ justify-content:space-between;
+  display: flex;
+}
+
+
+header  nav a {
+ text-decoration: none;
+    color: #b0b0b0;
+}
+
+header  nav a:hover {
+  text-decoration: underline;
+  color:black;
+}
+
+header .logo {
+    /*TRBL*/
+    margin: 28px 0 0 /*91px*/10.34% ;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 7.2rem;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    /*TRBL*/
+    margin: 0 /*82px*/9.31% 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 1.6rem;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top:/* 10px*/1.13% ;
+    cursor: pointer;
+}
+/* if I don't have this .cta img block here, get started button gets messy*/
+.cta img {
+    max-width: 100%;
+    margin-right: 1%;
+}
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 1.6rem;
+   line-height: 1.3rem;
+}
+
+.main-content .top-content {
+    display: flex;
+  justify-content: space-around;  
+}
+
+.main-content .middle-img {
+    max-width:100%;
+    display: block;
+    margin:/* 30px*/3.4%  auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-content .bottom-content p {
+    padding-right:/* 20px*/2.27% ;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+
+/* tablet*/
+@media (max-width:800px){
+    .container {
+    width: 100%;
+    background: orange;
+    
+  }
+  header {
+      flex-wrap: wrap;
+  }
+    header nav {
+        width: 100%;
+        padding: 38px 3%;
+           }
+   
+   header .logo {
+  display: block;
+  order: -1;
+  margin: 30px auto 0;
+}
+        .cta img{
+    display: none;
+       }
+
+       .cta {
+           justify-content: center;
+           margin-top: 35px;
+               }
+               .cta .cta-text h1 {
+                   letter-spacing: 0;
+                   margin-bottom: 15px;
+                                  }
```

Comment:
Between `.cta { ... }` and `.main-content .middle-img { ... }`, you can add `.cta .cta-text { margin: 0 }` to center your "Innovation On Demand" and button.